src/SkillsForm.jsx

```diff
@@ -16,23 +15,18 @@ class SkillForm extends React.Component {
     super(props);
     this.state = {
       skill_name: "",
-      skill_type: ""
+      skill_type: skillTypes[0].value
```

Comment:
Are you sure we want to default to "Compiled" type? I think that defaults should only be set for stuff that can reasonably be assumed to be valid for most clients, but I don't think that holds true in this case. It seems to me we should leave it without any defaults?